pilot/pkg/bootstrap/server.go

```diff
@@ -178,6 +186,7 @@ type PilotArgs struct {
 	MCPCredentialOptions *creds.Options
 	MCPMaxMessageSize    int
 	KeepaliveOptions     *istiokeepalive.Options
+	JWKSOptions					 JWKSOptions
```

Comment:
Please format to align with other fields.

Reply:
Done!